IMASampleApp/app/src/main/java/com/ooyala/sample/players/CustomConfiguredIMAPlayerActivity.java

```diff
@@ -32,6 +33,9 @@ public final static String getName() {
   final String PCODE  = "R2d3I6s06RyB712DN0_2GsQS-R-Y";
   final String DOMAIN = "http://ooyala.com";
 
+  // Write the sdk events text along with events count to log file in sdcard if the log file already exists
+  SDCardLogcatOoyalaEventsLogger playbacklog = new SDCardLogcatOoyalaEventsLogger();
```

Comment:
I don't think we have a rule that I know of, but personally my $0.02 is that the "new" should happen in the constructor, since this isn't a static final field.